src/plugins/ethereum/createIdentities.js

```diff
@@ -21,7 +20,7 @@ export function createIdentity(address: EthAddress): IdentityProposal {
 }
 
 export function createIdentities(
-  ethAddresses: JsonLog<EthAddress>
+  ethAddresses: $ReadOnlyArray<EthAddress>
```

Comment:
Same question: separate this change from the Flow upgrade?

Reply:
This was a latent bug uncovered by flow.

Reply:
I'll move this change into a PR that precedes this one.